modules/admin_manual/pages/configuration/server/occ_command/file_commands.adoc

```diff
@@ -230,7 +230,7 @@ TIP: If many users are affected, it could be convenient to create a shell script
 
 == The files:transfer-ownership command
 
-You may transfer all files and shares from one user to another. 
+You may transfer all files and shares from one user to another, including users who have never logged in. 
```

Comment:
```suggestion
You may transfer all files and shares from one user to another, including to a user who has never logged in yet. 
```
Maybe it will be clearer to say it like this ^
There is no point trying to transfer `from` a user who has never logged in, because that user will not have any files!

Reply:
It is clearer, yes. Thanks.